### PR TITLE
TIKE docs overhaul

### PR DESCRIPTION
**Pull request by ttdu**
To do:

- [x] Rewrite data-access NB
- [ ] Update package installation instructions
- [ ] Update quick-cloud-cutouts so that multiprocessing works again
- [ ] Fix broken links on intro.md
- [x] Eliminate links to MAST Labs

### CODE REVIEW

**Comment by ttdu** [comment]
Looks like the only mast-labs links were in the data-access NB, so good to go on this front

**Comment by ttdu** [comment]
Tagging @cmurray-astro for ease of reference

**Comment by ttdu** [comment]
@jaytmiller Ready for review/merge. Other tasks will be addressed in a future PR.

File: markdown/software-installed.md [hunk 1]
```diff
@@ -31,4 +34,6 @@ Commands which work in a terminal can also be used in a Jupyter notebook by pref
 
 ## Missing your favorite software?
 
-If a package you need is missing from the configuration, we encourage you to [open a GitHub issue](https://github.com/spacetelescope/jupyterhub-deploy/issues/new) or contact the [MAST help desk](mailto:archive@stsci.edu) to suggest the package for inclusion. Please include a brief justification explaining the audience and purpose of the software. See also: [How to install extra software?](extra-software.md)
+See also: [How to install extra software?](extra-software.md)
+
+If your favorite software should be part of the default configurationwe encourage you to [open a GitHub issue](https://github.com/spacetelescope/science-platform-images/issues) or contact the [MAST help desk](mailto:archive@stsci.edu) to suggest the package for inclusion. Please include a brief justification explaining the audience and purpose of the software.
```
Comment (jaytmiller) [inline markdown/software-installed.md:39]: Typo `configurationwe`

**Review by jaytmiller — APPROVED** [review]
This looks great.   Longer term I'm wondering about "software installed" and how it relates to multiple images.  Potentially there should be an image version note indicating e.g. "this info applies to image version 0.12.0".

**Pull request merged**